`get` command panics without second argument

The `get` command currently requires two arguments, a source path and destination path. Running it with one argument, e.g. `dbxcli get /dir/file`, throws this error:

```
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x4b2ce0, 0xc820010070)
	/home/travis/.gimme/versions/go1.6.2.linux.amd64/src/runtime/panic.go:481 +0x3e6
github.com/dropbox/dbxcli/cmd.get(0x7db940, 0xc820011200, 0x1, 0x1, 0x0, 0x0)
	/home/travis/gopath/src/github.com/dropbox/dbxcli/cmd/get.go:31 +0x59a
github.com/dropbox/dbxcli/vendor/github.com/spf13/cobra.(*Command).execute(0x7db940, 0xc8200111b0, 0x1, 0x1, 0x0, 0x0)
	/home/travis/gopath/src/github.com/dropbox/dbxcli/vendor/github.com/spf13/cobra/command.go:561 +0x62c
github.com/dropbox/dbxcli/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x7dd140, 0x7db940, 0x0, 0x0)
	/home/travis/gopath/src/github.com/dropbox/dbxcli/vendor/github.com/spf13/cobra/command.go:651 +0x55c
github.com/dropbox/dbxcli/vendor/github.com/spf13/cobra.(*Command).Execute(0x7dd140, 0x0, 0x0)
	/home/travis/gopath/src/github.com/dropbox/dbxcli/vendor/github.com/spf13/cobra/command.go:610 +0x2d
github.com/dropbox/dbxcli/cmd.Execute()
	/home/travis/gopath/src/github.com/dropbox/dbxcli/cmd/root.go:213 +0x23
main.main()
	/home/travis/gopath/src/github.com/dropbox/dbxcli/main.go:43 +0x14
```

In this situation I think it would be more intuitive to download the file into the current working directory, just like `sftp` and similar tools do.

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -28,8 +28,8 @@ import (
 )
 
 func get(cmd *cobra.Command, args []string) (err error) {
-	if len(args) < 1 {
-		return errors.New("source is required")
+	if len(args) == 0 || len(args) > 2 {
+		return errors.New("`get` requires 1 or 2 arguments")
 	}
 
 	src, err := validatePath(args[0])

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -37,11 +37,9 @@ func get(cmd *cobra.Command, args []string) (err error) {
 		return
 	}
 
-	var dst string
-	if len(args) < 2 {
-		// If a destination is not specified use the base of the source path
-		dst = path.Base(src)
-	} else {
+	// Default `dst` to the base segment of the source path; use the second argument if provided.
+	dst := path.Base(src)
+	if len(args) == 2 {
 		dst = args[1]
 	}
 

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -39,6 +39,7 @@ func get(cmd *cobra.Command, args []string) (err error) {
 
 	var dst string
 	if len(args) < 2 {
+		// If a destination is not specified use the base of the source path
 		dst = path.Base(src)
 	} else {
 		dst = args[1]

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -15,6 +15,7 @@
 package cmd
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"os"
@@ -27,11 +28,20 @@ import (
 )
 
 func get(cmd *cobra.Command, args []string) (err error) {
-	src, err := validatePath(args[0])
-	dst := args[1]
+	if len(args) < 1 {
+		return errors.New("source is required")
+	}
 
-	if dst == "" {
+	src, err := validatePath(args[0])
+	if err != nil {
+		return
+	}
+
+	var dst string
+	if len(args) < 2 {
 		dst = path.Base(src)
+	} else {
+		dst = args[1]
 	}
 
 	arg := files.NewDownloadArg(src)

--- a/cmd/get.go
+++ b/cmd/get.go
@@ -29,7 +29,7 @@ import (
 
 func get(cmd *cobra.Command, args []string) (err error) {
 	if len(args) == 0 || len(args) > 2 {
-		return errors.New("`get` requires 1 or 2 arguments")
+		return errors.New("`get` requires `src` and/or `dst` arguments")
 	}
 
 	src, err := validatePath(args[0])

--- a/cmd/put.go
+++ b/cmd/put.go
@@ -63,16 +63,14 @@ func put(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	src := args[0]
-	if err != nil {
-		return
-	}
 
-	var dst string
-	if len(args) < 2 {
-		// If a destination is not specified use the base of the source path
-		dst = "/" + path.Base(src)
-	} else {
+	// Default `dst` to the base segment of the source path; use the second argument if provided.
+	dst := "/" + path.Base(src)
+	if len(args) == 2 {
 		dst, err = validatePath(args[1])
+		if err != nil {
+			return
+		}
 	}
 
 	contents, err := os.Open(src)

--- a/cmd/put.go
+++ b/cmd/put.go
@@ -58,8 +58,8 @@ func uploadChunked(r io.Reader, commitInfo *files.CommitInfo, sizeTotal int64) (
 }
 
 func put(cmd *cobra.Command, args []string) (err error) {
-	if len(args) < 1 {
-		return errors.New("source is required")
+	if len(args) == 0 || len(args) > 2 {
+		return errors.New("`put` requires 1 or 2 arguments")
 	}
 
 	src := args[0]

--- a/cmd/put.go
+++ b/cmd/put.go
@@ -59,7 +59,7 @@ func uploadChunked(r io.Reader, commitInfo *files.CommitInfo, sizeTotal int64) (
 
 func put(cmd *cobra.Command, args []string) (err error) {
 	if len(args) == 0 || len(args) > 2 {
-		return errors.New("`put` requires 1 or 2 arguments")
+		return errors.New("`put` requires `src` and/or `dst` arguments")
 	}
 
 	src := args[0]

--- a/cmd/put.go
+++ b/cmd/put.go
@@ -15,9 +15,11 @@
 package cmd
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"os"
+	"path"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
 	"github.com/dustin/go-humanize"
@@ -56,10 +58,21 @@ func uploadChunked(r io.Reader, commitInfo *files.CommitInfo, sizeTotal int64) (
 }
 
 func put(cmd *cobra.Command, args []string) (err error) {
+	if len(args) < 1 {
+		return errors.New("source is required")
+	}
+
 	src := args[0]
-	dst, err := validatePath(args[1])
 	if err != nil {
 		return
+	}
+
+	var dst string
+	if len(args) < 2 {
+		// If a destination is not specified use the base of the source path
+		dst = "/" + path.Base(src)
+	} else {
+		dst, err = validatePath(args[1])
 	}
 
 	contents, err := os.Open(src)

